plugins/GSdx/GS.h

```diff
@@ -1414,3 +1414,13 @@ enum class GSVideoMode : uint8
 	DTV_720P,
 	DTV_1080I
 };
+
```

Comment:
maybe use flags?

```
enum class Filtering
{
	Nearest = 0x00,
	Bilinear = 0x01,
	Trilinear = 0x02,
	
	PS2 = 0x10,
	Forced = 0x20,
	Slow = 0x40,
	

	Bilinear_Forced = Bilinear | Forced,
	Bilinear_PS2  = Bilinear | PS2,
//	Trilinear_PS2 = Trilinear | PS2, //? is this the idea of Trilinear?
	Trilinear_Forced = Trilinear | Forced,
	Trilinear_Slow = Trilinear | Slow
};
```

Then the trilinear request in GSDeviceOGL::CreateSurface would be more simple (`(m_filter&Trilinear)==Trilinear)` or `(m_filter&Trilinear)>0`)